{ 'cebit_pose_greet': { 'head': [ { 'duration': 1.988,
                                    'name': u'cebit_pose_1_greet',
                                    'positions': [-0.3225, 0.0593],
                                    'starttime': 0.0}],
                        'left_arm': [ { 'duration': 1.988,
                                        'name': u'cebit_pose_1',
                                        'positions': [ -1.5398,
                                                       0.7699,
                                                       -0.2801,
                                                       0.0098,
                                                       0.02,
                                                       -0.2499,
                                                       0.8168],
                                        'starttime': 0.0}],
                        'left_leg': [ { 'duration': 1.988,
                                        'name': u'cebit_pose_1',
                                        'positions': [ 0.0041,
                                                       -0.0684,
                                                       -0.7712,
                                                       1.1807,
                                                       0.5958,
                                                       -0.0659],
                                        'starttime': 0.0}],
                        'right_arm': [ { 'duration': 1.988,
                                         'name': u'cebit_pose_1_greet',
                                         'positions': [ 1.5369,
                                                        1.4365,
                                                        1.0711,
                                                        -0.0102,
                                                        0.6042,
                                                        0.2615,
                                                        -1.5615],
                                         'starttime': 0.0}],
                        'right_leg': [ { 'duration': 1.988,
                                         'name': u'cebit_pose_1',
                                         'positions': [ -0.0041,
                                                        0.0684,
                                                        0.7712,
                                                        -1.1807,
                                                        -0.5958,
                                                        0.0659],
                                         'starttime': 0.0}],
                        'torso': [ { 'duration': 1.988,
                                     'name': u'cebit_pose_1',
                                     'positions': [0.0001, 0.0012],
                                     'starttime': 0.0}]}}
